test/e2e/vtc_routing_test.go

```diff
@@ -188,7 +188,7 @@ func TestVTCBasicRouting(t *testing.T) {
 
 		podHistogram := make(map[string]int)
 
-		for range 5 {
+		for i := 0; i < 5; i++ {
```

Comment:
new golang version can support range int. If your local env report issues. feel free to upgrade to v1.22 or above